.jenkins/sarus_run_test.sh

```diff
@@ -18,39 +20,36 @@ pip install -r ../requirements.txt
 python -c 'import fv3config; import yaml; fid=open("../tests/pytest/config/default.yml", "r"); config = yaml.safe_load(fid); fv3config.write_run_directory(config, "./c12_test")'
 deactivate
 
-cd ${PWD}/c12_test
+# Set the working directory for the upcoming SLURM jobs
+cd c12_test
 cp ../../.jenkins/job_jenkins_sarus .
-cp ../../.jenkins/md5*.txt .
+cp ../../tests/pytest/reference/circleci/default/md5.txt .
 export SCRATCH_DIR=${PWD}
 
 
-# Run a c12 regression test for each Docker image
-dockerfiles=( $( ls ../../docker/Dockerfile.* ) )
-for df in ${dockerfiles[@]}; do
-   # Grab the architecture tag from the dockerfile filename
-   arch=${df:24}
-
-   # Get the name of the Docker image and tar file
-   export FV3_CONTAINER=fv3gfs-compiled:${arch}
-   tar_file=fv3gfs-compiled_${arch}.tar
+# Run c12 regression test on each Docker image
+declare -a tags=("hpc" "hpc-serialize")
+for tag in ${tags}; do
+    # Copy archived version of the Docker image from a Google Storage Bucket
+    tar_file=fv3gfs-compiled-${tag}.tar
+    gsutil copy gs://vcm-jenkins/${tar_file}.gz .
+    gunzip ${tar_file}.gz
 
-   # Copy archived version of the Docker image from a Google Storage Bucket
-   gsutil copy gs://vcm-jenkins/${tar_file}.gz .
-   gunzip ${tar_file}.gz
+    # Load archive Docker image into the local Sarus container registry
+    export FV3_CONTAINER=fv3gfs-compiled:${tag}
+    module load sarus
+    sarus load ./${tar_file} ${FV3_CONTAINER}
+    module unload sarus
 
-   # Load archive Docker image into the local Sarus container registry
-   module load sarus
-   sarus load ./${tar_file} ${FV3_CONTAINER}
-   module unload sarus
+    # Launch SLURM job
+    sbatch --wait job_jenkins_sarus
 
-   # Launch SLURM job
-   sbatch --wait job_jenkins_sarus
+    # Verify results
+    md5sum -c ./md5.txt 
 
-   # Verify results
-   md5sum -c ./md5_${arch}.txt
+    # Clean up working directory
+    rm *.nc
```

Comment:
I was going to ask if would make sense to try to incorporate this into the test script and use that, but with the saur special requirements, probably not. Plus, you probably only want to test one case, not all+serialization cases. This is more of a touchstone/sanity check than reproducing all the tests.

Reply:
A next step probably is implementing the entire range of tests.  For now, I just wanted a preliminary test that runs and does some sort of validation.  But you are absolutely right in suggesting the full serialization/non-serialization list of tests